static/js/tabs.js

```diff
@@ -0,0 +1,44 @@
+function initTabs() {
```

Comment:
Hmm, I guess tabs is where no-js falls down.
I'd like to find a way to get this to work without JS - as with the homepage. Perhaps we can have the tabs load the page again, with sub-pages like:
`/charm/` -> Overview
`/charm/configuration` -> Configuration
etc.
Then when we add the JS, we can intercept these loads.